docs/CI-CD/GithubActions.md

```diff
@@ -1,8 +1,10 @@
 # Setup CI-CD with Github Actions
 
-Setup Github Actions to build and push the helium container to Azure Container Registry (ACR).
+Setup Github Actions to build and push the helium container to Azure Container Registry (ACR) and Docker hub.
```

Comment:
I really think we need to separate the instructions for ACR and docker hub. Our "customers" will seldom do "both". Combining the two makes it more complex for them.